tests/main.js

```diff
@@ -70,24 +64,31 @@ test.afterEach(() => {
 })
 
 // Convenience method for running a test for each JS bundler.
-const testBundlers = makeTestBundlers(test)
+const testMany = makeTestMany(test, {
+  bundler_default: {
+    nodeBundler: undefined,
+  },
+  bundler_esbuild: {
+    nodeBundler: ESBUILD,
+  },
+})
 
-testBundlers('Zips Node.js function files', [ESBUILD, ESBUILD_ZISI, DEFAULT], async (bundler, t) => {
+testMany('Zips Node.js function files', ['bundler_default', 'bundler_esbuild'], async (options, t) => {
```

Comment:
We're replacing `[ESBUILD, ESBUILD_ZISI, DEFAULT]` (3 entries) with `['bundler_default', 'bundler_esbuild']` (2 entries, `ESBUILD_ZISI` missing). It looks like we're effectively losing a third of our tests - am I missing smth?

Reply:
Great catch, thanks! Added back in https://github.com/netlify/zip-it-and-ship-it/pull/695/commits/0b187129cab43be9bf80ff1c10642a4e549bd0f4.